### PR TITLE
Fix component governance alerts

### PR DESCRIPTION
**Pull request by sebastienros**
###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/dotnet/aspire/pull/2049)

### CODE REVIEW

File: Directory.Packages.props [hunk 1]
```diff
@@ -120,5 +120,8 @@
     <PackageVersion Include="Microsoft.DotNet.Build.Tasks.Installers" Version="8.0.0-beta.23564.4" />
     <!-- unit test dependencies -->
     <PackageVersion Include="Microsoft.Extensions.Diagnostics.Testing" Version="8.0.0" />
+    <!-- Pinned version for Component Governance - Remove when root dependencies are updated -->
```
Comment (radical) [inline Directory.Packages.props:123]: Would it be useful to open an issue to remind us of doing that?

Comment (sebastienros) [reply Directory.Packages.props:123]: GitHub feature request: scheduled issues, we need a bot. "Hey GitHub, remind me to check the dependencies in one month"

Comment (mitchdenny) [reply Directory.Packages.props:123]: Sounds like a great little idea for a sample Aspire application that processes GitHub webhooks :)

**Review by JamesNK — APPROVED** [review]

**Review by mitchdenny — APPROVED** [review]

**Pull request merged**